### PR TITLE
added designs & system commands docs in 

### PR DESCRIPTION
**Pull request by parth721**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 98908e732b93d16a4fbdceb1b2bfbe8cfc606147 at: https://64bf8e7cf0febd41cc2d247e--meshery-docs-preview.netlify.app

File: docs/pages/guides/mesheryctl/mesheryctl_system.md
Comment (Boombag0607) [inline docs/pages/guides/mesheryctl/mesheryctl_system.md:1]: This file is redundant. See [here](https://docs.meshery.io/reference/mesheryctl).

**Comment by Chadha93** [comment]
@parth721  Let's discuss this on the Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

File: docs/pages/guides/mesheryctl/mesheryctl_system.md
Comment (leecalcote) [reply docs/pages/guides/mesheryctl/mesheryctl_system.md:1]: @suhail34 @alphaX86 will you offer direction here?

Comment (parth721) [reply docs/pages/guides/mesheryctl/mesheryctl_system.md:1]: Thanks @lee, happy learning 🙂.

**Comment by parth721** [comment]
Due to some reason I can revert this PR, so I am making a new PR & closing it. In the new One I added more specific mesheryctl system model commands, those are not present [here](https://docs.meshery.io/reference/mesheryctl) & fix the meshery-design file.

**Pull request closed without merging**